Libs/MRML/Core/vtkMRMLSliceNode.cxx

```diff
@@ -339,25 +379,22 @@ void vtkMRMLSliceNode::SetOrientation(const char* orientation)
     {
     return;
     }
- if (!strcmp(orientation, "Axial"))
-    {
-    this->SetOrientationToAxial();
-    }
-  else if (!strcmp(orientation, "Sagittal"))
-    {
-    this->SetOrientationToSagittal();
-    }
-  else if (!strcmp(orientation, "Coronal"))
-    {
-    this->SetOrientationToCoronal();
-    }
-  else if (!strcmp(orientation, "Reformat"))
-    {
-    this->SetOrientationToReformat();
-    }
-  else
+
+  if (!strcmp(orientation, "Reformat"))
+   {
+   this->SetOrientationToReformat();
+   return;
+   }
+
+  this->SliceToRAS->DeepCopy(this->GetSliceOrientationPreset(orientation));
```

Comment:
yeah I also noted it. I was pushing a commit, than I got more problem relative the allocation of the matrices